framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -154,7 +154,7 @@ qx.Mixin.define("qx.ui.core.MExecutable",
         // remove the old binding
         if (old != null && !old.isDisposed() && ids[property] != null)
         {
-          old.removeBinding(ids[property]);
+          this.removeBinding(ids[property]);
```

Comment:
Is now this used instead of old because you changed the binding direction here https://github.com/qooxdoo/qooxdoo/pull/9275/files#diff-3ef8954de08e91faabd2c8f4c7474e0cL180 ?